kubernetes-model/kubernetes-model-coordination/src/main/java/io/fabric8/kubernetes/api/model/coordination/v1/LeaseSpec.java

```diff
@@ -49,7 +49,7 @@
 @JsonDeserialize(using = com.fasterxml.jackson.databind.JsonDeserializer.None.class)
 @ToString
 @EqualsAndHashCode
-@Buildable(editableEnabled = false, validationEnabled = false, generateBuilderPackage = true, lazyCollectionInitEnabled = false, builderPackage = "io.fabric8.kubernetes.api.builder", inline = @Inline(type = Doneable.class, prefix = "Doneable", value = "done"))
+@Buildable(editableEnabled = false, validationEnabled = false, generateBuilderPackage = false, lazyCollectionInitEnabled = false, builderPackage = "io.fabric8.kubernetes.api.builder", inline = @Inline(type = Doneable.class, prefix = "Doneable", value = "done"))
```

Comment:
Why is this changed to false? Class generated with Annotator has it true

Reply:
Actually, I think this flag is for generating sundrio helper classes again. In `kubernetes-model-core`, we use KubernetesCoreTypeAnnotator which sets `generateBuilderPackage=true`, rest of the models are simply depending on `kubernetes-model-core` so we don't need to do it in this case. 

Earlier everything was under one module that's why earlier it's `generateBuilderPackage` was set to `true`

Reply:
These builders may actually be used by client users. If builders were being generated before, they should be still be generated. So `KubernetesTypeAnnotator` should set generateBuilderPackage=true too in order not to break compatibility.

Or is there any reason why this flag was set to false in `KubernetesTypeAnnotator`?

Reply:
By builder helper classes, I didn't mean builder itself. I meant these
![Screenshot from 2020-04-17 12-18-53](https://user-images.githubusercontent.com/13834498/79540367-a9813d80-80a5-11ea-9bd7-608cc5de26ef.png)

If you look at extensions' model they are using `kubernetes-model-core` and they have `generateBuilderPackage=false`. That way these classes won't be generated again and would be consumed from `kubernetes-model-core`
![Screenshot from 2020-04-17 12-19-04](https://user-images.githubusercontent.com/13834498/79540398-ba31b380-80a5-11ea-8e78-e2ce1ede9637.png)

Reply:
:+1:  nice explanation.
This may also help when dealing with the java 9 modularization.